.github/workflows/runtime-staging.yml

```diff
@@ -42,23 +44,21 @@ jobs:
       # - run: yarn test --ci --maxWorkers 15%
 
       - name: Deploy web-player
-        if: github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
         run: yarn deploy:web:staging
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_RUNTIME_KEY_STAGING }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_RUNTIME_SECRET_STAGING }}
 
       - name: Deploy native runtime
-        if: github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
         run: yarn deploy:staging
 
       - name: Notify Slack
         uses: 8398a7/action-slack@v3
-        if: always() && github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
```

Comment:
If you want to make the "Only from `main` or `sdk-` branches" foolproof, [we could use something like `startsWith(github.ref, 'refs/heads/sdk-')`](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#startswith)

Reply:
I think that's a good idea. Especially if we want to merge some of the workflows later on.
I've updated the `if` clauses accordingly. Let me know whether this looks good 👍

Reply:
LGTM!